src/BuiltInTools/dotnet-watch/LaunchSettingsProfile.cs

```diff
@@ -45,11 +47,32 @@ public class LaunchSettingsProfile
                 return null;
             }
 
+            if (string.IsNullOrEmpty(launchProfileName))
+            {
+                // Load the default (first) launch profile
+                return ReadDefaultLaunchProfile(launchSettings, reporter);
+            }
+
+            // Load the specified launch profile
+            var namedProfile = launchSettings?.Profiles?.FirstOrDefault(f => string.Equals(f.Key , launchProfileName, StringComparison.Ordinal)).Value;
```

Comment:
```suggestion
            var namedProfile = launchSettings?.Profiles?.FirstOrDefault(f => string.Equals(f.Key , launchProfileName, StringComparison.OrdinalIgnoreCase)).Value;
```
As this is something that the user types and I don't believe there's going to be many people (if any) with profiles differing only on casing.

Reply:
I had that originally but it turns out `dotnet run` is case sensitive, so we'd have to change it both places, likely to first do an ordinal search, then fallback to case-insensitive if not found.

Reply:
Given that dotnet run is case sensitive and we have a warning below I'm ok if we leave it as is. Might be worth pointing about the casing difference in the warning. It can be as simple as saying "Did you mean (suggestion)?" when we find a case insensitive match

Reply:
Yeah I like that idea.

Reply:
<img width="744" alt="image" src="https://user-images.githubusercontent.com/249088/172072346-883ae290-e192-4c1e-8f23-067119549c34.png">